internal/backend/muxes/muxes.go

```diff
@@ -2,4 +2,16 @@ package muxes
 
 import "net/http"
 
-type Muxes struct{ Auth, NoAuth *http.ServeMux }
+type (
```

Comment:
In general, “main” and “aux” are meaningless jargon, I don’t think anyone will see “aux” and think “ah! I know what this means/does!” or understand the logic of what is/isn’t “main”.

From an architecture perspective, we started with a single mux, then we duplicated it, then we merged the pair, and now we’re creating a tree of muxes - this is too complex to maintain by median developers, so that’s not healthy or safe.